core/src/io/anuke/mindustry/entities/effect/Lightning.java

```diff
@@ -131,4 +140,19 @@ public void draw(){
     public EntityGroup targetGroup(){
         return bulletGroup;
     }
+
+    protected static boolean hitsInsulator(Lightning l){
+        if (l.lines.size > 1){
+            Position from = l.lines.get(l.lines.size - 2);
+            Vector2 to = (Vector2) l.lines.get(l.lines.size - 1);
+
+            Tile a = world.tileWorld(from.getX(), from.getY());
```

Comment:
This can be null. Don't get the tile, just get its coordinates in world space.

Reply:
Forwarded its coordinates 🔮